src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c

```diff
@@ -542,12 +543,43 @@ void KaleidoScope_DrawWorldMap(PlayState* play, GraphicsContext* gfxCtx) {
         POLY_OPA_DISP = gfx;
     }
 
-#if OOT_DEBUG
     if (HREG(15) == 2) {
         HREG(15) = 1;
         HREG(14) = 6100;
         HREG(13) = 5300;
     }
+#else
+    gDPSetTextureFilter(POLY_OPA_DISP++, G_TF_POINT);
+
+    gDPLoadTLUT_pal256(POLY_OPA_DISP++, gWorldMapImageTLUT);
```

Comment:
is there no way to duplicate less code here?

Reply:
@fig02 ?

Reply:
I prefer as it is now personally, it takes a lot of mental energy and time for me to parse ifdeffed conditions and figure out what is actually happening.

If we want to do this in some extreme cases, maybe a middle ground is to have a comment describing the differences between debug and non-debug. but not sure

Reply:
I would really like to not duplicate those 30 lines, I think you should let the IDE work for you :'(

Oh well, if we add at least a comment to point out it's identical that's enough for me to not have to play 7 difference. Maybe:

```suggestion
#else
    // The following is identical to the code under the previous `HREG(15) == 0` branch under `OOT_DEBUG`

    gDPSetTextureFilter(POLY_OPA_DISP++, G_TF_POINT);

    gDPLoadTLUT_pal256(POLY_OPA_DISP++, gWorldMapImageTLUT);
```